src/libraries/System.Private.CoreLib/src/System/IO/FileStatus.SetTimes.OSX.cs

```diff
@@ -46,15 +53,19 @@ private static unsafe Interop.Error SetCreationTimeCore(string path, long second
             attrList.bitmapCount = Interop.libc.AttrList.ATTR_BIT_MAP_COUNT;
             attrList.commonAttr = Interop.libc.AttrList.ATTR_CMN_CRTIME;
 
-            Interop.Error error =
-                Interop.libc.setattrlist(path, &attrList, &timeSpec, sizeof(Interop.Sys.TimeSpec), new CULong(Interop.libc.FSOPT_NOFOLLOW)) == 0 ?
+            // Follow links when using SafeFileHandle API.
+            CULong flags = new CULong(handle is null ? Interop.libc.FSOPT_NOFOLLOW : 0);
```

Comment:
You can remove this variable, and its comment.
Instead, pass `new CULong(Interop.libc.FSOPT_NOFOLLOW)` directly for both `setattrlist` and `fsetattrlist`.

Reply:
You don't need to pass it for fsetattrlist as link resolution was already determined on open(), or am I missing something?

Reply:
> You don't need to pass it for fsetattrlist as link resolution was already determined on open(),

Yes, and because of that we expect `fsetattrlist` to have the same result whether `FSOPT_NOFOLLOW` is used or not.